### ISSUE TITLE
Missing some electron.abi.node in electron

### ISSUE THREAD

**Issue by penggelies07** [homebridge/node-pty-prebuilt-multiarch#18]
## Environment details

- OS: macOS (MacBook Air Apple M2)
- OS version: 13.0 
- node-pty version: 0.10.1-pre.5
- electron version: 21.4.4

## Issue description

When I execute npm i install is complete,  I run the program in my electron, and it reports an error:

![image](https://github.com/oznu/node-pty-prebuilt-multiarch/assets/20084372/46384183-9c4b-4cec-9af4-7fa8abbc71ec)

I print the log inside the code of node-pty-prebuilt-multiarch according to the error:

![image](https://github.com/oznu/node-pty-prebuilt-multiarch/assets/20084372/1b4f032c-8f10-48f0-99a4-aae471e551c1)

Below is what I print:

![image](https://github.com/oznu/node-pty-prebuilt-multiarch/assets/20084372/e4c57bcb-adb3-46c0-b1f0-8fa5ad5b9349)

I look in my node_modules folder:

![image](https://github.com/oznu/node-pty-prebuilt-multiarch/assets/20084372/4f07d447-54f4-497c-bb0a-133740c79655)

There is no electron-related version of abi.node at all!

I found the branch prebuilt-multiarch, there are some commits not published to the latest npm package?

Can you publish a usable npm package to support electron abi 109? Thank you so much~

**Comment by seydx** [comment]
> @seydx These electron versions ?
> 
> https://github.com/homebridge/node-pty-prebuilt-multiarch/blob/main/build.sh
> 
> target: 19, 20, 21, 22, 23, 24, 25, 26, 27 (latestVersion), 28 (beta)

Yeah this should cover all available versions (except of the nightly build 29)

**Comment by NorthernMan54** [comment]
@seydx I started an `electron` branch with the updated targets, but the build is failing starting at 20

```
prebuild info build Preparing to prebuild @homebridge/node-pty-prebuilt-multiarch@0.11.11 for electron 20.0.0 on linux-arm64 using node-gypusing libc musl
gyp info find Python using Python version 3.11.6 found at "/usr/bin/python3"
gyp http GET https://electronjs.org/headers/v20.0.0/node-v20.0.0-headers.tar.gz
gyp http 200 https://artifacts.electronjs.org/headers/v20.0.0/node-v20.0.0-headers.tar.gz?force_headers_dist=1
gyp http GET https://electronjs.org/headers/v20.0.0/SHASUMS256.txt
gyp http 200 https://artifacts.electronjs.org/headers/v20.0.0/SHASUMS256.txt?force_headers_dist=1
gyp info spawn /usr/bin/python3
gyp info spawn args [
gyp info spawn args   '/node-pty/node_modules/prebuild/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/node-pty/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/node-pty/node_modules/prebuild/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/tmp/prebuild/electron/20.0.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/tmp/prebuild/electron/20.0.0',
gyp info spawn args   '-Dnode_gyp_dir=/node-pty/node_modules/prebuild/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/tmp/prebuild/electron/20.0.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/node-pty',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build', '--jobs', 2 ]
make: Entering directory '/node-pty/build'
  CXX(target) Release/obj.target/pty/src/unix/pty.o
In file included from /tmp/prebuild/electron/20.0.0/include/node/v8-object.h:9,
                 from /tmp/prebuild/electron/20.0.0/include/node/v8-array-buffer.h:13,
                 from /tmp/prebuild/electron/20.0.0/include/node/v8.h:24,
                 from /tmp/prebuild/electron/20.0.0/include/node/node.h:69,
                 from ../node_modules/nan/nan.h:62,
                 from ../src/unix/pty.cc:20:
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:106:45: error: 'is_lvalue_reference_v' is not a member of 'std'; did you mean 'is_lvalue_reference'?
  106 |   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
      |                                             ^~~~~~~~~~~~~~~~~~~~~
      |                                             is_lvalue_reference
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:106:66: warning: logical not is only applied to the left hand side of comparison [-Wlogical-not-parentheses]
  106 |   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
      |                                                                  ^
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:106:67: error: template argument 1 is invalid
  106 |   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
      |                                                                   ^
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:106:71: error: expected unqualified-id before '>' token
  106 |   template <class U, std::enable_if_t<!std::is_lvalue_reference_v<U>>*>
      |                                                                       ^
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:123:43: error: 'is_lvalue_reference_v' is not a member of 'std'; did you mean 'is_lvalue_reference'?
  123 | template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
      |                                           ^~~~~~~~~~~~~~~~~~~~~
      |                                           is_lvalue_reference
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:123:64: warning: logical not is only applied to the left hand side of comparison [-Wlogical-not-parentheses]
  123 | template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
      |                                                                ^
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:123:65: error: template argument 1 is invalid
  123 | template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
      |                                                                 ^
/tmp/prebuild/electron/20.0.0/include/node/v8-maybe.h:123:70: error: expected unqualified-id before '=' token
  123 | template <class T, std::enable_if_t<!std::is_lvalue_reference_v<T>>* = nullptr>
      |                                                                      ^
../src/unix/pty.cc: In function 'void pty_after_waitpid(uv_async_t*)':
../src/unix/pty.cc:610:9: warning: 'void* memset(void*, int, size_t)' writing to an object of type 'class Nan::Persistent<v8::Function>' with no trivial copy-assignment [-Wclass-memaccess]
  610 |   memset(&baton->cb, -1, sizeof(baton->cb));
      |   ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../node_modules/nan/nan.h:411:
../node_modules/nan/nan_persistent_12_inl.h:12:40: note: 'class Nan::Persistent<v8::Function>' declared here
   12 | template<typename T, typename M> class Persistent :
      |                                        ^~~~~~~~~~
../src/unix/pty.cc: At global scope:
/tmp/prebuild/electron/20.0.0/include/node/node.h:862:7: warning: cast between incompatible function types from 'void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)' {aka 'void (*)(v8::Local<v8::Object>)'} to 'node::addon_register_func' {aka 'void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)'} [-Wcast-function-type]
  862 |       (node::addon_register_func) (regfunc),                          \
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/tmp/prebuild/electron/20.0.0/include/node/node.h:896:3: note: in expansion of macro 'NODE_MODULE_X'
  896 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
      |   ^~~~~~~~~~~~~
../src/unix/pty.cc:839:1: note: in expansion of macro 'NODE_MODULE'
  839 | NODE_MODULE(pty, init)
      | ^~~~~~~~~~~
make: Leaving directory '/node-pty/build'
make: *** [pty.target.mk:131: Release/obj.target/pty/src/unix/pty.o] Error 1
prebuild ERR! build error 
prebuild ERR! stack Error: `make` failed with exit code: 2
prebuild ERR! stack     at ChildProcess.onExit (/node-pty/node_modules/prebuild/node_modules/node-gyp/lib/build.js:203:23)
prebuild ERR! stack     at ChildProcess.emit (node:events:513:28)
prebuild ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
prebuild ERR! not ok 
prebuild ERR! build Error: `make` failed with exit code: 2
prebuild ERR! build     at ChildProcess.onExit (/node-pty/node_modules/prebuild/node_modules/node-gyp/lib/build.js:203:23)
prebuild ERR! build     at ChildProcess.emit (node:events:513:28)
prebuild ERR! build     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
Result  2 null undefined
```

**Comment by seydx** [comment]
https://github.com/homebridge/node-pty-prebuilt-multiarch/pull/34

**Comment by NorthernMan54** [comment]
Published as 0.11.12
